src/_common/content/components/media-item/media-item.ts

```diff
@@ -142,21 +167,19 @@ export default class AppContentMediaItem extends Vue {
 	}
 
 	created() {
-		this.owner.getHydrator().useData('media-item-id', this.mediaItemId.toString(), data => {
-			if (data) {
-				this.mediaItem = new MediaItem(data);
-			} else {
-				this.hasError = true;
-			}
-		});
+		this.mediaItem = createMediaItem(this.owner, this.mediaItemId);
+		if (!this.mediaItem) {
+			this.hasError = true;
+		}
 	}
 
 	mounted() {
 		this.computeSize();
+		this.contentViewerParent = findVueParent(this, AppContentViewer) as AppContentViewerTS;
```

Comment:
I think we want this to be optional because when editing the media items, they have no contentViewer parent - when I was doing `findRequiredVueParent`, things were throwing errors and breaking because of this.

Changing to `findVueParent<AppContentViewerTS>(this, AppContentViewer);`